components/build/jvm-build-service/kustomization.yaml

```diff
@@ -164,5 +164,8 @@ patches:
           memory: 8Gi
           cpu: "2"
 
+patchesStrategicMerge:
+  - operator-images.yaml
```

Comment:
`JVM_DELETE_TASKRUN_PODS` is being updated on line 94, that should be removed, otherwise it's twice in final env.